src/Symfony/Component/HttpFoundation/Exception/JsonException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Exception;
+
+class JsonException extends \RuntimeException implements RequestExceptionInterface
```

Comment:
IMHO, this exception class should be final.
```suggestion
final class JsonException extends \RuntimeException implements RequestExceptionInterface
```

Reply:
Am I right that by implementing `RequestExceptionInterface` we automatically give it an HTTP 400 status code?
According to https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Exception/RequestExceptionInterface.php#L17

Reply:
> Am I right that by implementing `RequestExceptionInterface` we automatically give it an HTTP 400 status code?

Yes that is correct: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpKernel/HttpKernel.php#L69-L71

Reply:
Awesome, thnx!